test/test.mjs

```diff
@@ -941,6 +941,8 @@ async function startBrowser({ browserName, headless, startUrl }) {
       "dom.events.asyncClipboard.clipboardItem": true,
       // It's helpful to see where the caret is.
       "accessibility.browsewithcaret": true,
+      "layout.accessiblecaret.enabled": true,
+      "layout.accessiblecaret.hide_carets_for_mouse_input": false,
```

Comment:
Is there a way to pass these two options just for one specific test?

Reply:
I don't know.
@whimboo, do you know if it's possible ?
If not, it'd be nice to have such a feature.

Reply:
So the only way is to open a new browser, this isn't ideal but it's the only solution we've.

Reply:
Yeah, until we have support for chrome scope this is the only solution. The launcher.spec.ts unit tests in Puppeteer have some examples for that as well in case you need some more details.